### PR TITLE
[Translations] ProgressBar: i18n support

### PR DESCRIPTION
**Pull request by HalvorHaugan**
### Component Checklist 📝

- [x] JSDoc
- [x] Examples
- [x] Documentation
- [x] Storybook
- [x] Style mappings (`@navikt/core/css/config/_mappings.js`)
- [x] Component tokens (`@navikt/core/css/tokens.json`)
- [x] CSS class deprecations (`@navikt/aksel-stylelint/src/deprecations.ts`)
- [x] Exports (`@navikt/core/react/src/index.ts` and `@navikt/core/react/package.json`)
- [x] New component? CSS import (`@navikt/core/css/index.css`)
- [x] Breaking change? Update migration guide. Consider codemod.
- [x] Changeset (Format: `<Component>: <gitmoji?> <Text>.` E.g. "Button: :sparkles: Add feature xyz.")

### CODE REVIEW

File: @navikt/core/react/src/util/i18n/locales/nb.ts [hunk 1]
```diff
@@ -44,4 +44,9 @@ export default {
       labelSuffix: "slett",
     },
   },
+  ProgressBar: {
+    progress: "{value} av {valueMax}",
+    progressUnknown:
+      "Framdrift kan ikke beregnes, antatt tid er {seconds} sekunder.",
```
Comment (KenAJoh) [inline @navikt/core/react/src/util/i18n/locales/nb.ts]: These are both equivalent, but "fremdrift" feels more correct (??)
```suggestion
      "Fremdrift kan ikke beregnes, antatt tid er {seconds} sekunder.",
```

File: @navikt/core/react/src/util/i18n/locales/nb.ts [hunk 2]
```diff
@@ -44,4 +44,9 @@ export default {
       labelSuffix: "slett",
     },
   },
+  ProgressBar: {
+    progress: "{value} av {valueMax}",
```
Comment (KenAJoh) [inline @navikt/core/react/src/util/i18n/locales/nb.ts]:
```suggestion
    progress: "{currentValue} av {valueMax}",
```

File: @navikt/core/react/src/progress-bar/ProgressBar.tsx [hunk 3]
```diff
@@ -130,17 +139,13 @@ export const ProgressBar = forwardRef<HTMLDivElement, ProgressBarProps>(
       >
         <div
           className={cl("navds-progress-bar__foreground", {
-            "navds-progress-bar__foreground--indeterminate": Number.isInteger(
-              simulated?.seconds,
-            ),
+            "navds-progress-bar__foreground--indeterminate": simulated?.seconds,
```
Comment (KenAJoh) [inline @navikt/core/react/src/progress-bar/ProgressBar.tsx]: @larseirikhansen Will this break anything? Should 0 seconds be a valid entry here?

File: @navikt/core/react/src/util/i18n/locales/nb.ts (hunk 2 above)
Comment (HalvorHaugan) [reply @navikt/core/react/src/util/i18n/locales/nb.ts]: What do you think about just writing "current" and "max"?

File: @navikt/core/react/src/util/i18n/locales/nb.ts (hunk 1 above)
Comment (HalvorHaugan) [reply @navikt/core/react/src/util/i18n/locales/nb.ts]: I think this is just a matter of what you are used to. For example, Morten likes the "fram" variant iirc. It doesn't matter much to me, as long as we are consistent.

File: @navikt/core/react/src/util/i18n/locales/nb.ts (hunk 2 above)
Comment (KenAJoh) [reply @navikt/core/react/src/util/i18n/locales/nb.ts]: Sounds good 👍

File: @navikt/core/react/src/progress-bar/ProgressBar.tsx (hunk 3 above)
Comment (HalvorHaugan) [reply @navikt/core/react/src/progress-bar/ProgressBar.tsx]: My change broke possibility to set it to 0 seconds. Fixed now. I've used `!== undefined` b.c. `isInteger()` returns false on decimals.

**Review by KenAJoh — DISMISSED** [review]

**Pull request merged**